### PR TITLE
Update /managed secondary nav

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Update secondary nav `Overview`, `Kafka`, `Cassandra` to `Overview`, `OpenStack`, `Kubernetes`, `Apps` 
- Add `Kafka`, `Cassandra` to `Apps`' secondary nav
- Redirect `/managed/kafka` to `/managed/apps/kafka`
- Redirect `/managed/Cassandra` to `/managed/apps/Cassandra`
- Fix the 404 error on `/managed/contact-us`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/managed
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please check page against [copy doc](https://docs.google.com/document/d/1fLRWW-Gtslaia2ypSrLJVBxZVHo5FY5Q84o97Gs2-w4/edit).

## Issue / Card

Fixes [#4370](https://github.com/canonical-web-and-design/web-squad/issues/4370#event-5199410812)

## Screenshots

image
image

### CODE REVIEW

**Comment by minkyngkm** [comment]
It doesn't seem any button goes to the `/managed/contact-us` page when it's clicked. It just opens modal.

**Comment by grobbie** [comment]
@minkyngkm /managed/contact-us is linked to from blogs and white papers

File: templates/managed/apps/cassandra.html [hunk 1]
```diff
@@ -139,7 +139,7 @@ <h2 class="p-muted-heading u-align--center">Managed on:</h2>
         <h3>
           Upstream Cassandra on-rails, fully automated and integrated into your infrastructure.
         </h3>
-        <p><a href="https://ubuntu.com/managed#get-in-touch" class="js-invoke-modal">Get started with a free architecture assessment&nbsp;&rsaquo;</a></p>
+        <p><a href="/managed/contact-us" class="js-invoke-modal">Get started with a free architecture assessment&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/managed/apps/cassandra.html:142]: Nice spots :+1:

File: redirects.yaml [hunk 2]
```diff
@@ -571,7 +571,9 @@ livepatch/?: "/security/livepatch"
 lotus/?: "/ibm"
 lxd/?: "https://linuxcontainers.org/"
 maas/?: "https://maas.io/"
-managed-apps/?: "/managed"
+managed/kafka/?: "/managed/apps/kafka"
+managed/cassandra?: "/managed/apps/cassandra"
+managed-apps/?: "/managed/apps"
```
Comment (anthonydillon) [inline redirects.yaml:576]: Missing a redirect from `/managed/apps/contact-us` to `/managed/contact-us`

**Comment by minkyngkm** [comment]
> @minkyngkm /managed/contact-us is linked to from blogs and white papers

@grobbie Oh thanks for letting me know :)

**Review by anthonydillon — APPROVED** [review]
Code and QA :+1: with one missing redirect to add.

File: redirects.yaml (hunk 2 above)
Comment (minkyngkm) [reply redirects.yaml:577]: Thanks for reviewing! I've added it now! :)

**Pull request merged**